dao/src/main/java/se/fortnox/reactivewizard/db/transactions/Transaction.java

```diff
@@ -2,68 +2,41 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import rx.Observable;
 import se.fortnox.reactivewizard.db.ConnectionProvider;
-import se.fortnox.reactivewizard.db.statement.Statement;
 
 import java.sql.Connection;
 import java.sql.SQLException;
-import java.util.Collection;
-import java.util.HashSet;
 import java.util.LinkedList;
-import java.util.Set;
+import java.util.List;
 import java.util.concurrent.ConcurrentLinkedQueue;
 import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.atomic.AtomicReference;
 
 public class Transaction<T> {
 
     private static final Logger                                      log                 = LoggerFactory.getLogger(Transaction.class);
-    private final        AtomicReference<ConnectionProvider>         connectionProvider  = new AtomicReference<>();
-    private final        Iterable<Observable<T>>                     daoCalls;
-    private final        ConcurrentLinkedQueue<TransactionStatement> statementsToExecute = new ConcurrentLinkedQueue<>();
-    private final        Set<TransactionStatement>                   statementsToSubscribe = new HashSet<>();
+    private final        ConnectionProvider                          connectionProvider;
+    private final        ConcurrentLinkedQueue<TransactionStatement> statementsToExecute;
     private final        AtomicBoolean                               waitingForExecution = new AtomicBoolean(true);
```

Comment:
I think you can drop this since we do no longer support asynchronous subscription of the DaoObservables.